doc/release-notes/0.38/0.38.md

```diff
@@ -90,15 +90,15 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
 
 <tr>
 <td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/13767">#13767</a></td>
-<td valign="top">Compressed references mode is not available</td>
+<td valign="top">Compressed references mode is not available.</td>
 <td valign="top">Apple silicon macOS</td>
 <td valign="top">You can use only the large heap (non-compressed references) mode.</td>
 <td valign="top">None</td>
 </tr>
 
 <tr>
 <td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/14803">#14803</a></td>
-<td valign="top">Using the <tt>-XX:+ShowHiddenFrames</tt> option in an OpenJ9 release that is built with OpenJDK 18 and later causes errors due to underlying problems in OpenJDK</td>
+<td valign="top">Using the <tt>-XX:+ShowHiddenFrames</tt> option in an OpenJ9 release that is built with OpenJDK 18 and later causes errors due to underlying problems in OpenJDK.</td>
```

Comment:
Another "due to underlying problems in OpenJDK" here

Reply:
Removed.

Reply:
This phrase is also used in older release notes: 0.32, 0.35 and 0.36. Should they also be updated?

Reply:
The history is in https://github.com/eclipse-openj9/openj9-docs/issues/910. We could say the option is incompatible with OpenJDK 18 and later.

Reply:
@pshipton The PR is merged. Is this still a pending change: state that the option is incompatible?

Reply:
It's not a big deal, not worth the effort. I did put the release notes into the 0.37 branch, which I don't usually do, before tagging the release. It seemed a good idea in this case. We won't be able to make any more updates to in the 0.37 branch, but could still update the head stream if something important comes up. The doc and release page link to the head stream.